bcda/client/bluebutton.go

```diff
@@ -176,6 +178,34 @@ func (bbc *BlueButtonClient) GetCoverage(beneficiaryID, jobID, cmsID, since stri
 	return bbc.getBundleData(u, jobID, cmsID, nil)
 }
 
+func (bbc *BlueButtonClient) GetClaim(mbi, jobID, cmsID, since string, transactionTime time.Time) (*models.Bundle, error) {
```

Comment:
let's pass in the ClaimsWindow to these functions as well and set the service-date based on it.  I know the current search functions in BFD will ignore it, but if we do it now it will eventually work once we complete DCGEO-117